src/store/radix-api.ts

```diff
@@ -2239,6 +2239,16 @@ export type ApplicationAlias = {
   environmentName: string;
   url: string;
 };
+export type DnsAliasStatus = {
+  Condition?: string;
+  Message?: string;
+};
+export type DnsAlias = {
+  Status?: DnsAliasStatus;
```

Comment:
API needs to be regenerated, Status must be lowercase

Reply:
fixed